assets/dfn-panel.css

```diff
@@ -4,20 +4,36 @@ dfn {
 }
 
 .dfn-panel {
+  --fill: #fff;
   position: absolute;
   left: var(--left); /* set via JS */
   top: var(--top); /* set via JS */
   z-index: 35;
-  height: auto;
-  width: max-content;
-  max-width: 300px;
-  max-height: 500px;
```

Comment:
If we want overflow with max-height, then we would need to add a wrapper to dfn-panel to display the caret. Otherwise, as both caret and panel are `position: absolute`, the panel hides the caret.

Reply:
let's go with this for now. If we need to, we can modify later.